tests/network/vmi_networking.go

```diff
@@ -522,6 +513,8 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 	})
 
 	Context("VirtualMachineInstance with masquerade binding mechanism", func() {
+		const LibvirtDirectMigrationPort = 49152
```

Comment:
Unrelated question: Is this port still relevant?

Reply:
I see it defined in production and in the tests, so I would say it is relevant.
But I have not got into the details too much.